### ISSUE TITLE
LSTM training error

### ISSUE THREAD

**Issue by MihirBha** [srivatsan88/End-to-End-Time-Series#6]

**Comment by srivatsan88** [comment]
I see you are using fit method rather fit_generator. One is you can go back to 2.2 TF version and use fit_generator. I will try to change the code against latest version and update

**Comment by srivatsan88** [comment]
Hi, I was able to make the same code work on TF 2.5 as well. instead of fit_generated which is deprecated fit works as well. I am hardening the code and will upload in upcoming week

This is what I used in TF 2.5

early_stopping = tf.keras.callbacks.EarlyStopping(monitor='val_loss',
                                                    patience=2,
                                                    mode='min')

model.compile(loss=tf.losses.MeanSquaredError(),
              optimizer=tf.optimizers.Adam(),
              metrics=[tf.metrics.MeanAbsoluteError()])

history = model.fit(train_generator, epochs=20,
                    validation_data=val_generator,
                    shuffle=False,
                    callbacks=[early_stopping])

**Comment by MihirBha** [comment]
Hi sir ,
I am Mihir Bhawsar. I work on LSTM model to forecast next 15 days value .
I am using your code but I can't predict next 15 days value.
So please help me to my project☺️.
